content/applications/sales/sales/products_prices/products/variants.rst

```diff
@@ -122,12 +101,41 @@ once an attribute is added to a product.
 .. warning::
    Once added to a product, an attribute's :guilabel:`Variants Creation Mode` cannot be edited.
 
-Lastly, the :guilabel:`eCommerce Filter Visibility` field determines whether these attribute options
-are visible to the customer on the front-end, as they shop on the online store.
+In, the :guilabel:`eCommerce Filter Visibility` field determines whether these attribute options are
+visible to the customer on the front-end, as they shop on the online store.
 
 - :guilabel:`Visible`: the attribute values are visible to customers on the front-end.
 - :guilabel:`Hidden`: the attribute values are hidden from customers on the front-end.
 
+Lastly, in the optional :guilabel:`eCommerce Category` field, select a category from a drop-down
+menu to group similar attributes under the same section for added specificity and organization.
```

Comment:
You could link to a product categoyr page/doc section (if one exists), or just explain where this list is pulled from.

Reply:
I totally understand the suggestion, and I believe the note admonition that follows leads the reader to that exact information. 👍